plugins/defaultplugins/ifplugin/data_resync.go

```diff
@@ -980,6 +980,13 @@ func (plugin *InterfaceConfigurator) registerInterface(ifName string, ifIdx uint
 	if err := vppcalls.SetInterfaceTag(ifName, ifIdx, plugin.vppCh, plugin.Stopwatch); err != nil {
 		return fmt.Errorf("error while adding interface tag %s, index %d: %v", ifName, ifIdx, err)
 	}
+	// Add AF-packet type interface to local cache
+	if ifData.Type == intf.InterfaceType_AF_PACKET_INTERFACE {
+		if plugin.Linux != nil && plugin.afPacketConfigurator != nil  && ifData.Afpacket != nil {
```

Comment:
Minor detail, but after second condition you have two spaces.. do you auto format code on save?